lib/cn/csched_sp3.c

```diff
@@ -1470,7 +1483,7 @@ sp3_comp_thread_name(
     else
         node_type = 'i';
 
-    snprintf(buf, bufsz, "%c%s_%s_%u%u", node_type, a, r, loc->node_level, loc->node_offset);
+    snprintf(buf, bufsz, "hse_%c%s_%s_%u%u", node_type, a, r, loc->node_level, loc->node_offset);
```

Comment:
Isn't this used for pthread_setname?  Which only supports 16 chars?  Seems like a waste to use 4 for "hse_" given that "ps" will make it clear that this thread belongs to the app using HSE.

Reply:
Handy when trolling through gdb output:

$ sudo pstack $(pidof kmt) |egrep 'Thread.*hse_' |egrep -v '(hse_cn|hse_wal|hse_kvdb|hse_throt|hse_log|hse_timer|hse_pfx|hse_lc|hse_c0sk|hse_ctxn|hse_url|hse_kmc|foo])'
Thread 141 (Thread 0x7e71877fa000 (LWP 7055) "hse_xkc_lg_2130"):
Thread 139 (Thread 0x7e6d173fd000 (LWP 7053) "hse_rsp_sr_00"):
Thread 138 (Thread 0x7e7454bfb000 (LWP 7052) "hse_isp_si_18"):
Thread 137 (Thread 0x7e745bfff000 (LWP 7051) "hse_rsp_sr_00"):
Thread 136 (Thread 0x7e74a13fb000 (LWP 7038) "hse_xsp_b1_111"):
Thread 133 (Thread 0x7e74563fa000 (LWP 7027) "hse_isp_si_10"):
Thread 132 (Thread 0x7f87a8d99000 (LWP 7026) "hse_xkc_lg_2136"):
Thread 131 (Thread 0x7e74553fc000 (LWP 7019) "hse_isp_si_13"):
Thread 128 (Thread 0x7e786abff000 (LWP 6762) "hse_isp_si_19"):
Thread 127 (Thread 0x7f87a7d97000 (LWP 6761) "hse_xkc_lg_2135"):
Thread 123 (Thread 0x7e74997fb000 (LWP 6747) "hse_xkc_lg_2140"):
Thread 122 (Thread 0x7e74bbfff000 (LWP 6746) "hse_xsp_b1_112"):
Thread 115 (Thread 0x7f8903fde000 (LWP 6731) "hse_xkc_lg_2133"):
Thread 12 (Thread 0x7f89017d9000 (LWP 6572) "hse_sp3_monitor"):
Thread 11 (Thread 0x7f8902fdc000 (LWP 6569) "hse_xsp_b1_110"):
Thread 2 (Thread 0x7f891f7a5000 (LWP 6542) "hse_jclock"):